.github/workflows/release.yml

```diff
@@ -16,6 +16,14 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Generate Release Notes
+        id: generate_notes
+        uses: conventional-changelog/releaser-action@v1
```

Comment:
Nice! Just linking this issue for completeness sake, touch I don't think it should affect us: https://github.com/TriPSs/conventional-changelog-action/issues/239